Frameworks/UIKit/StarboardXaml/CompositorInterface.h

```diff
@@ -167,7 +167,7 @@ class DisplayTextureXamlGlyphs : public DisplayTexture {
 
     float _desiredWidth, _desiredHeight;
     void Measure(float width, float height);
-    void ConstructGlyphs(const char* fontName, const wchar_t* str, int len);
+    void ConstructGlyphs(Microsoft::WRL::Wrappers::HString fontFamilyName, const wchar_t* str, int len);
```

Comment:
This needs to be a const & otherwise it's pass by Val which will copy the inner string